src/ProjectSettings.cpp

```diff
@@ -98,38 +95,10 @@ void ProjectSettings::UpdatePrefs()
 #endif
 }
 
-void ProjectSettings::SetSnapTo(int snap)
-{
-   mSnapTo = snap;
-}
-   
-int ProjectSettings::GetSnapTo() const
-{
-   return mSnapTo;
-}
-
 void ProjectSettings::SetTool(int tool) {
    if (auto oldValue = mCurrentTool; oldValue != tool) {
       mCurrentTool = tool;
       Notify( mProject, ChangedTool, oldValue );
    }
 }
 
-static ProjectFileIORegistry::AttributeWriterEntry entry {
-[](const AudacityProject &project, XMLWriter &xmlFile){
-   auto &settings = ProjectSettings::Get(project);
-   xmlFile.WriteAttr(wxT("snapto"),
-                     settings.GetSnapTo() ? wxT("on") : wxT("off"));
-}
-};
-
-static ProjectFileIORegistry::AttributeReaderEntries entries {
-// Just a pointer to function, but needing overload resolution as non-const:
-(ProjectSettings& (*)(AudacityProject &)) &ProjectSettings::Get, {
-   // PRL:  The following has persisted as a per-project setting for long.
-   // Maybe that should be abandoned.  Enough to save changes in the user
-   // preference file.
-   { "snapto", [](auto &settings, auto value){
-      settings.SetSnapTo(value.ToWString() == wxT("on") ? true : false);
-   } },
-} };
```

Comment:
As we mentioned today -- moving ALL of the persistency code into the green, if my other PRs for track subclasses are merged too.  Good milestone.